lib/pure/collections/intsets.nim

```diff
@@ -7,698 +7,18 @@
 #    distribution, for details about the copyright.
 #
 
-## The ``intsets`` module implements an efficient `int` set implemented as a
-## `sparse bit set`:idx:.
-##
-## **Note**: Currently the assignment operator ``=`` for ``IntSet``
-## performs some rather meaningless shallow copy. Since Nim currently does
-## not allow the assignment operator to be overloaded, use `assign proc
-## <#assign,IntSet,IntSet>`_ to get a deep copy.
-##
+## Deprecated by the generic `OrdSet` for ordinal sparse sets.
```

Comment:
CI fails:

Error: undeclared field: 'id' for type system.int [type declared in /Users/runner/work/Nim/Nim/lib/system/basic_types.nim(2, 3)]